js1/uvod-do-js/vypis-do-stranky.md

```diff
@@ -20,26 +20,19 @@ Mějme následující stránku.
 </html>
 ```
 
-Když ji otevřeme v prohlížeči, můžeme si v konzoli ověřit, co proměnná :var[document.body.innerHTML] obsahuje.
+V proměnné :var[document.body.innerHTML] je pak uložen řetězec:
 
-```jscon
-> document.body.innerHTML
+```js
 "
     <h1>První program</h1>
 
 "
 ```
 
-Obdrželi jsme řetězec s celým HTML stránky včetně všech mezer a odsazení. Pokud chceme obsah stránky změnit, můžeme prostě změnit obsah této proměnné a prohlížeč se už sám postará o překreslení stránky. Vyzkoušejte si to v konzoli.
+Všiměte si, že řetězec obsahuje i všechny mezery a odsazení.
 
-```jscon
-> document.body.innerHTML = '<h1>Nový obsah</h1>';
-```
-
-Tímto postupem nahradíme celý obsah stránky obsahem našeho řetězce. Nejčastěji však budeme chtít k obsahu stránky pouze něco přidat. K tomu využijeme operátor `+=`.
+Pokud chceme obsah stránky změnit, můžeme prostě změnit obsah této proměnné a prohlížeč se už sám postará o překreslení stránky. Například můžete k řetězci v :var[document.body.innerHTML] přičíst další řetězec, klidně i třeba s kouskem HTML.
 
 ```jscon
-> document.body.innerHTML += '<p>Výsledek výpočtu</p>';
+> document.body.innerHTML = '<h1>Nový obsah</h1>';
 ```
```

Comment:
Tenhle snippet už asi nechápu. Proč je to `jscon`? A proč se nad ním mluví o `+=` („přičíst další řetězec“), když tady je `=`?